weeks/week07/week07.md

```diff
@@ -1,3 +1,125 @@
 # Week 7: Web Automation with Playwright
 
-Coming soon.
+
+## Summary:
+
+Playwright is the powerful tool for Web Test automation (UI and API) in different programming languages:
+- TypeScript, 
+- JavaScript,
+- .NET,
+- Java.
+- Python
+
+It is 
+- Cross-browser,
+- Cross-platform,
+- Support mobile testing
+
+The scope of this page: Playwright - Python
+
+
+## Playwright - Python General information:
+
+1. Convenient for Web and API testing
+2. Intuitive integration with Pytest Test Automation Framework
+3. Intuitive integration with Allure
+4. Inherited  support for all Python functionality
+5. Easy to use within CI/CD (e.g. Jenkins)
+
+
+## References:
+
+1. [Playwright - Python oficial page](https://playwright.dev/python/)
+2. [Installation](https://playwright.dev/python/docs/intro)
+3. [Pytest fixtures](https://docs.pytest.org/en/6.2.x/fixture.html#autouse-fixtures-fixtures-you-don-t-have-to-request)
+
+
+## Playwright and Pytest:
```

Comment:
We don't need to talk about `pytest` here as it is in the Week 5 (Testing Frameworks).